bundle-core/src/main/java/net/discdd/pathutils/TransportPaths.java

```diff
@@ -0,0 +1,26 @@
+package net.discdd.pathutils;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.nio.file.Path;
+
+@Getter
+@Setter
```

Comment:
we don't want setters because once the paths are initialized, we don't want them to change, and we probably don't want getters either since the fields are public.